src/content/topics/integrations/more/compass.mdx

```diff
@@ -0,0 +1,144 @@
+
+## Prerequisites
+
+To connect Compass to LaunchDarkly, you must meet the following prerequisites:
+
+1. Be a Compass administrator on your Atlassian site
+2. Generate a LaunchDarkly [API access token](/home/account-security/api-access-tokens) for use with Compass.
+* Create a token with the **Reader** role.
+* If you are on an Enterprise plan, check the **This is a service token** option when creating your acesss token.
+* If you are on a Pro plan, leave this option unchecked.
+* Be sure to copy the new token to your clipboard.
+
+## Setting up the Compass integration
+
+The Compass integration is managed from the Compass product on your Atlassian Cloud site. Other than creating an API access token, there are no Compass settings or preferences to configure in LaunchDarkly.
+
+1. Generate a LaunchDarkly API access token as described in the [Prerequisites](#prerequisites) section.
+2. Go to your Atlassian Compass site, e.g. https://mysite.atlassian.net/compass.
+3. In the top nagivation, click the **Apps** link.
+4. Find **LaunchDarkly** in the list of apps and click the **Install** button.
+5. After the LaunchDakly app is installed, click the **Configure** button.
+6. Click the **Allow access** button to approve the access to your Atlassian account requested by the LaunchDarkly app.
+7. On the **LaunchDarkly is requesting Access** screen, review the requested permissions, then click the **Accept** button.
+
+<Callout intent="info">
+<CalloutTitle>You must authorize the requested permissions for the integration to work</CalloutTitle>
+<CalloutDescription>
+
+If you click **Cancel**, the integration cannot access your Compass data.
+
+</CalloutDescription>
+</Callout>
+
+![LaunchDarkly authorization screen for Compass](compass-ld-integration-auth-dialog.png)
+
+8. On the app configuration screen, paste your LaunchDarkly API access token and click the **Submit** button.
+
+![LaunchDarkly access token screen for Compass](compass-ld-integration-access-token.png)
+
+9. Confirm that the integration has connected LaunchDarkly and Compass when the **Connected to LaunchDarkly** success message is displayed.
+
+![Successfully connected Compass to LaunchDarkly](compass-ld-integration-connected.png)
+
+## Connecting Compass components to LaunchDarkly
+
+Now that you've set up the Compass integration for Compass, you must conncet your LaunchDarkly projects to Compass components in order to see feature flag data in Compass.
+
+To connect a LaunchDarkly project to a Compass component:
+
+1. In LaunchDarkly, navigate to the **Feature flags** page for the project you want to connect.
+2. Copy the project URL, e.g. https://app.launchdarkly.com/starship-command/production/features, to your clipboard.
+3. Navigate to your Compass site, e.g. https://mysite.atlassian.net/compass
+4. Navigate to the Compass component you want to connect to LaunchDarkly.
+5. In the **Other links** section of your Compass component, paste the LaunchDarkly project URL as a new link.
```

Comment:
Per [this conversation](https://launchdarkly.slack.com/archives/C01HCMH33JM/p1649017618030749) we should update this line to be specific about linking to the project and environment.